GooglePlayInstant/Editor/PlayInstantAssetBundleBrowserClient.cs

```diff
@@ -15,28 +15,34 @@
 using System;
 using System.IO;
 using System.Linq;
+using System.Text.RegularExpressions;
 using UnityEditor;
 using UnityEngine;
 
 namespace GooglePlayInstant.Editor
 {
     public static class AssetBundleBrowserClient
     {
-        private const string AssetBundleBrowserMenuItem = "Window/AssetBundle Browser";
         private static bool? _assetBundleBrowserIsPresent;
-
+        private const string AssetBundleBrowserName = "com.unity.assetbundlebrowser";
+        private const string AssetBundleBrowserMenuItem = "Window/AssetBundle Browser";
+        public static readonly string AssetBundleBrowserVersion = GetBrowserVersion();
```

Comment:
Resolved

For when version is updated and reducing the performance cost, I made changes such that computations for detecting asset bundle browser and its version are done only when the "Create Bundle" tab is clicked. This way, the developer can just close and reopen quick deploy window, or simply open another tab and come back to this one for the changes to be reflected.

Got  a question tho! @thanwah I'm Wondering if we could just add  a small refresh button instead of recomputing. This would reduce the cost of computation even more, assuming that the developer will rarely update the installed version.